Likvido.Kubesec/Likvido.Kubesec/Utils.cs

```diff
@@ -23,10 +24,16 @@ public static void WriteToFile(string file, IReadOnlyList<Secret> secrets, strin
             streamWriter.WriteLine($"# Secret: {secretsName}");
             streamWriter.WriteLine("#######################################");
 
-            foreach (var secret in secrets)
-            {
-                streamWriter.WriteLine($"{secret.Name}={secret.Value}");
-            }
+            var serializer = new YamlDotNet.Serialization.Serializer();
+
+            //hack due to https://github.com/aaubry/YamlDotNet/issues/361
+            // trying to keep "\n" in kubernetes but Environment.NewLine locally
+            var content = serializer
+                .Serialize(secrets.ToDictionary(s => s.Name, s => s.Value.Replace(Environment.NewLine, "\n")))
+                .Replace(Environment.NewLine, "\n")
+                .Replace("\n", Environment.NewLine);
```

Comment:
This looks a bit weird. Why would you change it back and forth?

Reply:
if for some reason a content will have \r\n already I don't want to end up with \r\r\n

Reply:
ahh, got it